data/serializer/src/main/java/tech/pegasys/teku/api/response/v1/EventType.java

```diff
@@ -17,7 +17,7 @@
 import java.util.stream.Collectors;
 
 @SuppressWarnings("JavaCase")
-public enum EventType {
+public enum EventType { // TODO move logic outside of here?
```

Comment:
probably optional at this point, the eventType will have to live on for now so it's not too horrible...